### DISCUSSION TITLE
tpotce  data

### DISCUSSION THREAD

**Discussion by kevinelwell** [telekom-security/tpotce#895] [Q&A] [answered]
When I reboot tpotce, the only data in the dashboards that seems to persist is the Suricata data. Is this by design? If so, is there a way to make all the data persist?

**Comment by t3chn0m4g3** [comment, marked as answer]
No, but you did set a timeframe other than 15 minutes in the upper right corner?

**Reply by kevinelwell** [reply]
Yes, I went back 30 days. I see all of the Suricata data, but that is about it.

**Reply by t3chn0m4g3** [reply]
All detected events are persisted for [90 days](https://github.com/telekom-security/tpotce/blob/master/etc/curator/actions.yml).
If you do not even see any other data than Suricata than it is most likely your machine was either not exposed correctly or just did not have any events. You can run the `/opt/tpot/bin/hptest.sh` to verify your installation is working. However, afterwards you should reboot the machine.

**Reply by kevinelwell** [reply]
I ran `/opt/tpot/bin/hptest.sh` and it installed some missing components like nmap, netcat, and others. I am rebooting now. Thank you.

**Reply by t3chn0m4g3** [reply]
It only installs the components needed to run the test. If all is successful you will see some events in Kibana.

**Reply by kevinelwell** [reply]
It was successful. After the reboot, i see data. Thank you for your assistance.
